### DISCUSSION TITLE
Different images for light and dark modes

### DISCUSSION THREAD

**Discussion by kjelljorner** [pradyunsg/furo#131] [Ideas] [answered]
First of all I would like to congratulate on an awesome sphinx theme! I really love the automatic change between light and dark modes as well. For logos in the navigation menu, it is easy to setup up different images for light and dark modes. For other figures I found myself resorting to a type of [hack](https://rhyslloyd.me/serve-dark-mode-images-natively/). I would basically insert this html code with a `.. raw:: html` directive in my .rst file. Not sure if this could be handled in a better way?

```html
  <picture>
      <source srcset="_static/logo-dark.svg" media="(prefers-color-scheme: dark)">
      <img src="_static/logo-light.svg?text=Light+mode" /> 
  </picture>
```

**Comment by pradyunsg** [comment, marked as answer]
You can set the "only-dark" and "only-light" classes on those images:

```rst
.. image:: https://source.unsplash.com/200x200/daily?cute+dogs
   :align: center
   :class: only-light

.. image:: https://source.unsplash.com/200x200/daily?cute+cats
   :align: center
   :class: only-dark
```

**Reply by pradyunsg** [reply]
This reminded me that I never documented this! Fixed that in https://github.com/pradyunsg/furo/commit/aaec3694e26a0cfe7226fbb40e00e50a8771e7ab. :)

**Reply by gumityolcu** [reply]
Hello. Currently, this doesn't work for inline images. Adding `:class: only-light` will add a newline before and after the image when used inside a substitution as an inline image. Is there any fix for this? Thank you
